reladomo/src/main/java/com/gs/fw/common/mithra/attribute/DateAttribute.java

```diff
@@ -139,6 +139,16 @@ public void setSqlParameter(int index, PreparedStatement ps, Object o, TimeZone
         }
     }
 
+    @Override
+    public String formattedValue(Object object)
+    {
+        if (object == null)
+        {
+            return "null";
+        }
+        return "\"" + object + "\"";

```

Comment:
I think the null check may not be necessary. I wanted to make sure I never add the quoted string "null" nor throw.